autoPlay.js

```diff
@@ -128,10 +130,46 @@ function goToLaneWithBestTarget() {
 			}
 		}
 	
+		//Prefer lane with raining gold, unless current enemy target is a treasure or boss.
+		if(lowTarget != ENEMY_TYPE.TREASURE && lowTarget != ENEMY_TYPE.BOSS ){
```

Comment:
lowTarget isn't a monster type, it's the slot in the current lane to target. This will malfunction.